packages/perseus/src/widgets/interactive-graphs/graphs/quadratic.tsx

```diff
@@ -34,16 +34,6 @@ export function QuadraticGraph(props: QuadraticGraphProps) {
     const y = (x) => (a * x + b) * x + c;
 
     const handleOnMove = (destination: vec.Vector2, elementId: number) => {
-        // If the destination is invalid, we want do not want to move the point
-        const validDestination = isValidDestination(
-            destination,
-            elementId,
-            coords,
-        );
-        if (validDestination === false) {
-            return;
-        }
-
         dispatch(movePoint(elementId, destination));
     };
```

Comment:
Since `handleOnMove` is just calling `dispatch` now, I wonder if its definition could be inlined into the JSX. Of course, we might have to undo that later if we want to optimize things with `useCallback`. Not a big deal either way.